src/components/profile/ProfileService.js

```diff
@@ -8,6 +8,48 @@ goog.require('ga_urlutils_service');
     'pascalprecht.translate'
   ]);
 
+  module.filter('indicator', function() {
```

Comment:
I meant you can  use the filter which already exist : https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/measure/MeasureService.js#L7